spring-cloud-task-stream/pom.xml

```diff
@@ -42,7 +42,13 @@
 		</dependency>
 		<dependency>
 			<groupId>org.springframework.cloud</groupId>
-			<artifactId>spring-cloud-stream-test-support</artifactId>
+			<artifactId>spring-cloud-stream-test-support-internal</artifactId>
+			<version>1.0.0.BUILD-SNAPSHOT</version>
+			<scope>test</scope>
+		</dependency>
+		<dependency>
+			<groupId>org.springframework.cloud</groupId>
```

Comment:
The TaskEventTests should be skipped if Redis is not available.  At this time they fail, if Redis is not running.